subprojects/solver/solver-smtlib/src/main/java/hu/bme/mit/theta/solver/smtlib/impl/generic/GenericSmtLibSolverBinary.java

```diff
@@ -51,6 +51,11 @@ public GenericSmtLibSolverBinary(final Path solverPath, final String[] args,
         solverProcessBuilder.setProcessListener(processHandler);
 
         solverProcess = solverProcessBuilder.start();
+        try {
+            Thread.sleep(1000);
```

Comment:
Can we set this lower?

Reply:
Set to 50, it's hard to find the best value. We always have the chance that the process dies slower than that